client/mysqlbinlog.cc

```diff
@@ -2904,6 +2944,11 @@ static Exit_status dump_local_log_entries(PRINT_EVENT_INFO *print_event_info,
       my_close(fd, MYF(MY_WME));
       return ERROR_STOP;
     }
+
+    /* record the successful open in progress log */
+    if (progress_file != NULL)
+      fprintf(progress_file, "# OPEN %s with starting position %lu\n", logname, (unsigned long) start_position_mot);
```

Comment:
no ulongs, please. File offsets are not ulongs, they are ulonglongs.  You can cast to ulonglong, if your gcc is picky and use %llu as format string